src/components/RoomNameInput.js

```diff
@@ -108,7 +96,7 @@ class RoomNameInput extends Component {
                 prefixCharacter={CONST.POLICY.ROOM_PREFIX}
                 placeholder={this.props.translate('newRoomPage.social')}
                 onChange={this.setModifiedRoomName}
-                value={this.state.roomName.substring(1)}
+                defaultValue={this.props.initialValue}
```

Comment:
@eVoloshchak sorry that I didn't notice this earlier - 

1. Passing `initialValue` shows an extra `#`.
    We could fix this by not displaying the first character of `initialValue` (gotta be careful about the edge cases tho). Please let me know if you have any better ideas to fix this.

2. I believe we can remove `initialValue` prop from WorkspaceNewRoomPage.

Reply:
> 1. Passing `initialValue` shows an extra `#`.
>    We could fix this by not displaying the first character of `initialValue` (gotta be careful about the edge cases tho). Please let me know if you have any better ideas to fix this.

Yup, my bad, omitting first character was actually implemented before. 
> 2. I believe we can remove `initialValue` prop from WorkspaceNewRoomPage.

We can, updated the PR

Reply:
Thanks!